client/my-sites/domains/domain-management/components/domain/transfer-flag.jsx

```diff
@@ -34,6 +34,9 @@ class DomainTransferFlag extends PureComponent {
 			case transferStatus.PENDING_OWNER:
 				message = translate( 'Email Confirmation Required' );
 				break;
+			case transferStatus.PENDING_START:
+				message = translate( 'Action Required' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **45** times:
`translate( 'Action required!' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).